streams/th.m3u

```diff
@@ -82,56 +76,46 @@ http://183.89.246.119:8881/play/a0a6/index.m3u8
 http://183.89.246.119:8881/play/a0ad/index.m3u8
 #EXTINF:-1 tvg-id="MCOTHD.th",MCOT HD
 https://cdn6.goprimetime.info/feed/202306140918/chmcothd/index.m3u8
-#EXTINF:-1 tvg-id="MCOTHD.th",MCOT HD (1080p)
-http://183.89.246.119:8010/play/a00h/index.m3u8
 #EXTINF:-1 tvg-id="MCOTHD.th",MCOT HD (720p)
 https://cdn6.goprimetime.info/feed/202306140918/chmcothd3/index.m3u8
 #EXTINF:-1 tvg-id="MediaTV.th",Media TV (576p)
 http://183.89.246.119:8881/play/a0af/index.m3u8
 #EXTINF:-1 tvg-id="Mono29.th",Mono 29 (576p)
-http://183.89.246.119:8010/play/a00q/index.m3u8
+https://cdn6.goprimetime.info/feed/202306140918/chmono293/index.m3u8
 #EXTINF:-1 tvg-id="NationTV.th",Nation TV (576p)
-http://183.89.246.119:8010/play/a00o/index.m3u8
-#EXTINF:-1 tvg-id="NBT2HD.th",NBT 2 HD (1080p)
-https://thainews.prd.go.th/lv/live/ch1_L.sdp/playlist.m3u8
-#EXTINF:-1 tvg-id="NBT11Central.th",NBT 11 Central (1080p)
-https://thainews.prd.go.th/lv/live/ch9_L.sdp/playlist.m3u8
+https://cdn6.goprimetime.info/feed/202306140918/chnation3/index.m3u8
 #EXTINF:-1 tvg-id="NBT2HD.th",NBT HD (720p) [Not 24/7]
-https://cdn6.goprimetime.info/feed/chnbt3/index.m3u8
-#EXTINF:-1 tvg-id="NBTWorld.th",NBT World (720p)
-https://thainews.prd.go.th/lv/live/ch1_L_L.sdp/playlist.m3u8
+https://cdn6.goprimetime.info/feed/202306140918/chnbt3/index.m3u8
 #EXTINF:-1 tvg-id="One31.th",One 31 (1080p)
-http://183.89.246.119:8010/play/a00e/index.m3u8
+https://cdn6.goprimetime.info/feed/202306140918/chonehd3/index.m3u8
 #EXTINF:-1 tvg-id="PoliceTV.th",Police TV (1080p)
 https://cdn-th-vip.livestreaming.in.th/policetv/policetv/playlist.m3u8
+#EXTINF:-1 tvg-id="PPTVHD.th",PPTV HD
```

Comment:
First of all, the `tvg-id` is wrong: https://iptv-org.github.io/channel?id=PPTVHD36.th
Secondly, the channel is on the [blocklist](https://github.com/iptv-org/database/blob/master/data/blocklist.csv) because of the claims of the Bundesliga owner: https://github.com/github/dmca/blob/master/2020/09/2020-09-16-dfl.md

<img width="1417" alt="image" src="https://github.com/iptv-org/iptv/assets/7253922/661bc70a-7ff6-491c-87cd-cc62a4922e55">